google-cloud-storage/src/test/java/com/google/cloud/storage/it/GrpcPlainRequestLoggingInterceptor.java

```diff
@@ -223,6 +245,118 @@ private static ByteString snipBytes(ByteString content) {
     return content.substring(0, 20).concat(snip);
   }
 
+  // while trailers.get can return null, we're always getting values based on the keys it told us
```

Comment:
I don't quite follow this comment; could you clarify it a bit?

Reply:
rewrote the comment to be more clear. Let me know if you it is sufficient now.